blog/_posts/2015-07-17-Meet-Team-RailsGEnthusiasts.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Introducing Team RailsGEnthusiasts
+layout: blogpost
+created_at: Fri July 17 2015
+permalink: blog/_posts/2015-07-17-Meet-Team-RailsGEnthusiasts.md
+current: blog
+author: Boryana & Sophia
+twitter: RailsGEnthusia
+---
+
+
+###The Beginning of a Wonderful Journey
+
+Hello to everyone from Sofia, Bulgaria :)<br/> 
+We are Sophia and Boryana - Team RailsGirlsEnthusiasts (RailsGEnthusiasts)
+and our couches Andrew R., Marta R., Tsvety M. :)
+
+###Who are we?
+<img src="https://cloud.githubusercontent.com/assets/9285851/8754798/bf350628-2cce-11e5-9d0a-74ba118ec0d9.jpg"><br/>
```

Comment:
could you add all images to the `img` folder directly as explained in the [readme](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/README.md)? This is to ensure that there will be no broken images after publishing :)